### PR TITLE
Feature: Show welcome page on extension installation

### PR DESCRIPTION
**Pull request by hassan-pullflow**
### Summary of Changes 

- created command `Pullflow: Welcome`
- created `welcome.ts` in `webviews`
- created `welcome.css` `webviews/welcome`
- created `welcomeView.ts` in `commands`
- added `pullflow-logo.png` in assets
- updated code to show welcome page on first activation

### Snap-Shots

Screenshot 2023-10-31 at 12 07 39 AM

Screenshot 2023-10-31 at 12 07 57 AM

Screenshot 2023-10-31 at 12 08 10 AM

### Review-Tips

Start with: `extension.ts`

### PR-Specific-Tests

- User installs extension > User sees welcome page.
- User runs command `Pullflow: Welcome` > User sees welcome page.

### CODE REVIEW

**Review by amna-pullflow — APPROVED** [review]

File: src/views/webviews/welcome/welcome.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: Arial, sans-serif;
```
Comment (srzainab) [inline src/views/webviews/welcome/welcome.css:2]: Can we use our own fonts?

File: src/views/webviews/welcome/welcome.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+header {
+  margin-top: 20px;
```
Comment (srzainab) [inline src/views/webviews/welcome/welcome.css:6]: should we be hardcoding pixels? What happens when we resize the window, etc?

File: src/views/webviews/welcome/welcome.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  font-size: 1.0em;
+}
+
+#intro p {
+  color: #c2c9d3;
+  line-height: 1.5;
+}
+
+#intro strong {
+  color: #ffffff;
```
Comment (srzainab) [inline src/views/webviews/welcome/welcome.css:44]: Is there some type of theme that we can use instead of hardcoding colours?

**Review by srzainab — APPROVED** [review]

File: src/views/webviews/welcome/welcome.css (hunk 1 above)
Comment (hassan-pullflow) [reply src/views/webviews/welcome/welcome.css:2]: Our fonts don't look good on page

File: src/views/webviews/welcome/welcome.css (hunk 3 above)
Comment (hassan-pullflow) [reply src/views/webviews/welcome/welcome.css:44]: Will set theme once we have more web views and have proper frame work for them

File: src/views/webviews/welcome/welcome.css (hunk 2 above)
Comment (hassan-pullflow) [reply src/views/webviews/welcome/welcome.css:6]: I have checked resizing of window, vscode web view automatically adjusts pixels based on it.

**Pull request merged**